src/polyswarm_api/__init__.py

```diff
@@ -1073,25 +1030,6 @@ def lookup_uuids(self, uuids):
         """
         return self.loop.run_until_complete(self.ps_api.lookup_uuids(uuids))
 
-    def post_file(self, file_obj, filename):
```

Comment:
Looking at things, I know why these are exposed now. Without them, a user has no way in the synchronous API to submit a file(s) without blocking on the result. For now, I would like to keep these exposed. Sorry for the misleading info.

Reply:
Added the single artifact methods back, keeping compatibility.